src/_guides/language/language-tour.md

```diff
@@ -282,19 +289,28 @@ Any variable can have a constant value.
 
 <?code-excerpt "misc/lib/language_tour/variables.dart (const-vs-final)"?>
 {% prettify dart %}
-// Note: [] creates an empty list.
-// const [] creates an empty, immutable list (EIL).
-var foo = const []; // foo is currently an EIL.
-final bar = const []; // bar will always be an EIL.
-const baz = []; // baz is a compile-time constant EIL.
+var foo = const [];
+final bar = const [];
+const baz = []; // Equivalent to `const []`
+{% endprettify %}
+
+You can omit `const` from the initializing expression of a `const` declaration,
+like for `baz` above. For details, see [DON’T use const redundantly][].
 
-// You can change the value of a non-final, non-const variable,
-// even if it used to have a const value.
-foo = [];
+You can change the value of a non-final, non-const variable,
+even if it used to have a const value:
 
-// You can't change the value of a final or const variable.
-// bar = []; // Unhandled exception.
-// baz = []; // Unhandled exception.
+<?code-excerpt "misc/lib/language_tour/variables.dart (reassign-to-non-final)"?>
+{% prettify dart %}
+foo = [1, 2, 3]; // Was const []
+{% endprettify %}
+
+Being compile-time constants, you can't assign to a const variable:
```

Comment:
How about:

> While a const variable must be initialized, you can't assign to it at runtime:

Reply:
"While" might be confusing. It'd be clearer to say "you can't change the value of a const value", which is true but the implementation goes further (you can't assign it the same value, either).

But maybe we should change the example to so that it _does_ use a different value... That's how most people would mess up, isn't it?